libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -214,10 +214,13 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
     });
 
     private getTrialCountdown = (expiredDate: string) => {
-        const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date()) + 1; // include expired date.
-        if (trialDaysLeft === 1) {
+        const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date()) - 1; // subtract expired date
```

Comment:
I've thought again and maybe we don't have to subtract 1 like this. Please make the following change instead:

```
const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date()) - 1; // subtract expired date
```

↓

```
const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date());
```